apiary.apib

```diff
@@ -2618,6 +2618,124 @@ Your Accredible account is represented as an Issuer within the API.
             }
         }
 
+# Group Data Exports
+
+## Data Exports [/v1/data_exports]
+
+### View All Data Exports [GET]
+
+Returns list of data exports of departments accessible via the used API_KEY.
+
++ Request (application/json)
+
+    + Headers
+            Authorization: Token token=YOUR API_KEY
+
++ Response 200 (application/json)
+
+        {
+           "data_exports": [
+                    {
+                        "id": 1,
+                        "organization_id": 2,
```

Comment:
we don't use `organization_id` for the Public API endpoint, please use `department_id`